src/Stratis.Bitcoin/Connection/PeerBanning.cs

```diff
@@ -105,7 +107,12 @@ public bool IsBanned(IPEndPoint endpoint)
             PeerAddress peerAddress = this.peerAddressManager.FindPeer(endpoint);
 
             if (peerAddress == null)
+            {
+                this.logger.LogTrace("(-)[PEERNOTFOUND]");
                 return false;
+            }
+
+            this.logger.LogTrace("(-)");
```

Comment:
Here and on line 111, the exit log should contain the return value, so for example here it should look like:

```
    this.logger.LogTrace("(-):{0}", res);
```

On line 111

```
    this.logger.LogTrace("(-)[PEER_NOT_FOUND]:false");
```